v3/src/components/web-view/web-view.scss

```diff
@@ -0,0 +1,44 @@
+@import "../vars.scss";
+
+.codap-web-view-body {
+  background-color: white;
+  height: calc(100% - $title-bar-height);
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  .codap-web-view-backdrop {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    position: absolute;
+    transform: rotate(-30deg) translateY(35%);
```

Comment:
This confused me initially -- perhaps a comment about it being used for the loading error message.